scheme/avro/src/main/java/cz/o2/proxima/scheme/avro/AvroSerializerFactory.java

```diff
@@ -102,13 +102,12 @@ private Schema getAvroSchemaForClass(String avroClassName) {
           Class<? extends SpecificRecord> avroClass =
               Classpath.findClass(avroClassName, SpecificRecord.class);
           Method method = avroClass.getMethod("getSchema");
-          return (Schema) method.invoke(avroClass.newInstance());
+          return (Schema) method.invoke(Classpath.newInstance(avroClass));
```

Comment:
This is non relevant change. Just fixed one deprecation warning : )

Reply:
:+1: